### DISCUSSION TITLE
Como construir um modelo de ML capaz de prever possíveis resultados da copa do mundo de 2022

### DISCUSSION THREAD

**Discussion by mlarasusan** [gittogethers/octogatosconf22#116] [Talks in Portuguese]
**Speaker**: Luis Vinicius
**Assunto**: Será abordado como foi construído e como é possível construir um modelo capaz de prever possíveis resultados e vencedor da copa do mundo de 2022 utilizando python e suas bibliotecas voltadas para ciência de dados, e a extração dos dados da FIFA. 
 
Primeiramente, na palestra será apresentado como foi realizada a extração dos dados da FIFA com python e quais dados foram utilizados para a construção do modelo de Machine Learning. Após isso, será abordado como foi realizado o desenvolvimento do modelo de machine learning e a escolha do algoritmo para realizar a previsão dos possíveis resultados, também será falado um pouco como é calculado os possíveis resultados através do algoritmo e por fim apresentar quais são os resultados que o modelo conseguiu prever. Ao final, se sobrar tempo, será apresentado como foi realizado o deploy do modelo e construção de aplicação para o modelo ficar disponível na web e ser utilizado por outras pessoas que queiram saber os possíveis resultados que o modelo consegue prever. 

- Aplicação: https://possiveis-resultados-qatar2022.herokuapp.com/

**Comment by luislauriano** [comment]
Repositório do projeto: https://github.com/luislauriano/Data_Science/tree/master/Possiveis%20resultados%20copa%20do%20mundo%202022
